pkg/controller/kubelet-config/kubelet_config_controller.go

```diff
@@ -360,6 +360,18 @@ func generateOriginalKubeletConfigWithFeatureGates(cc *mcfgv1.ControllerConfig,
 		return nil, fmt.Errorf("could not merge feature gates: %v", err)
 	}
 
+	// (TODO) KubeletConfig can be modified based on the cgroupMode.
+	// modifying the kubelet's node-status-update-frequency based on the WorkerLatencyProfile
+	if cc.Spec.Node != nil {
+		switch cc.Spec.Node.Spec.WorkerLatencyProfile {
+		case configv1.MediumUpdateAverageReaction:
+			originalKubeConfig.NodeStatusUpdateFrequency = metav1.Duration{Duration: 20 * time.Second}
```

Comment:
Instead of hard coding here, create a const variable.

Reply:
These consts should probably be in the API as consts.

Reply:
https://github.com/openshift/api/pull/1136 PR has been raised for the same.